### PR TITLE
Requirements cleanup + Update CI pipelines

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
Converting again to draft as i'm adding a few things.

**Review by pmac — APPROVED** [review]
Looks good. I'm assuming this one is based on the commit in #1477 so I guess they're both good to go 👍

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -9,51 +9,58 @@ stages:
   - test
   - deploy
 
-build-images:
+build-image-django:
```
[the remark this answers is not in the document]
Comment (glogiotatidis) [reply .gitlab-ci.yml:12]: Actually #1477 is the first commit in this one. I built on top b/c I though it would conflict.

**Comment by glogiotatidis** [comment]
Tests passed here https://gitlab.com/mozmeao/snippets-service/-/pipelines/223577096

**Pull request merged**